x/stake/client/cli/tx.go

```diff
@@ -211,26 +211,11 @@ func GetCmdDelegate(cdc *codec.Codec) *cobra.Command {
 	return cmd
 }
 
-// GetCmdRedelegate implements the redelegate validator command.
```

Comment:
Idk, I think the original godoc made more sense, no?

Reply:
ah yes, it did

Reply:
Yes! 🙊 messed this one up @alexanderbez

Reply:
Ohhh well. Quick trigger finger on this one.